datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -43,17 +45,26 @@ def test_get_disposition_filename():
     eq_(len(output), 1)
     eq_(output[0]['filename'], 'T1.nii.gz')
 
+
 def test_assign():
-    raise SkipTest('TODO')
-    raise SkipTest('TODO with interpolations')
+    data = {'x': 'y'}
+
+    gen = assign({'key': 'value', 'aa': 'bb %(x)s'}, interpolate=True)
+    eq_(list(gen(data)), [{'key': 'value', 'aa': 'bb y', 'x': 'y'}])
+
+    genf = assign({'key': 'value %(x)s'}, interpolate=False)
+    eq_(list(genf(data)), [{'key': 'value %(x)s', 'x': 'y'}])
+
 
 def test_rename():
     raise SkipTest('TODO')
 
+
```

Comment:
Yes, teach @yarikoptic about blank lines, required by PEP8! ;)
Oh, and: Welcome to datalad!